### PR TITLE
[full-ci] Wording: Rename Quicklink to link

### PR DESCRIPTION
**Pull request by lookacat**
## Description
We've renamed "Quicklink" to "link" in the UI
See https://github.com/owncloud/web/issues/7891

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7891

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by JanAckermann — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/35087/65/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingPublicBasic-publicLinkCreate_feature-L165.png
  


</p></details>

**Review by kulmann — APPROVED** [review]

**Comment by kulmann** [comment]
PR needs a rebase please :-)

File: tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature [hunk 1]
```diff
@@ -172,10 +172,10 @@ Feature: Create public link shares
       | uid_owner   | Alice          |
       | permissions | read           |
       | path        | /simple-folder |
-      | name        | Quicklink      |
+      | name        | Link           |
     And the following success message should be displayed on the webUI
```
Comment (SwikritiT) [inline tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature:176]: due to this change the test is failing with latest ocis https://github.com/owncloud/web/issues/8929. The then step asserts from an api call. So if I understand this correctly. The naming from `QuickLink` to `link` is just on the UI level and API stills responds with `Quicklink`?

Comment (JammingBen) [reply tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature:176]: Yes, that was just a UI change.

**Pull request merged**